source/api/image/3.0/index.md

```diff
@@ -477,143 +477,116 @@ Servers should also support [CORS][cors-response] on Image Information responses
+| `@context` | Required | The context document that describes the semantics of the terms used in the document. This must be the URI `http://iiif.io/api/image/{{ page.major }}/context.json` |
 | `id` | Required | The base URI of the image as defined in [URI Syntax][uri-syntax], including scheme, server, prefix and identifier without a trailing slash. |
 | `type` | Required | The type for the Image API. The value _MUST_ be the string `ImageService3`. |
 | `protocol` | Required | The URI `http://iiif.io/api/image` which can be used to determine that the document describes an image service which is a version of the IIIF Image API. |
+| `profile` | Required | A string indicating the highest [compliance level][compliance-levels] which is fully supported by the service. The value _MUST_ be one of "level0", "level1", or "level2". |
 | `width` | Required | The width in pixels of the full image content, given as an integer. |
 | `height` | Required | The height in pixels of the full image content, given as an integer. |
-| `profile` | Required | A list of profiles, indicated by either a URI or an object describing the features supported. The first entry in the list _MUST_ be a [compliance level URI][compliance-levels]. |
-| `sizes` | Optional | A set of JSON objects with the `height` and `width` properties. These sizes specify preferred values to be provided in the `w,h` syntax of the size request parameter for scaled versions of the complete image.  In the case of servers that do not support requests for arbitrary sizes, these may be the only sizes available. A request constructed with the `w,h` syntax using these sizes _MUST_ be supported by the server, even if arbitrary width and height are not. |
-| `tiles` | Optional | A set of descriptions of the parameters to use to request regions of the image (tiles) that are efficient for the server to deliver. Each description gives a width, optionally a height for non-square tiles, and a set of scale factors at which tiles of those dimensions are available. |
+| `maxWidth`  | Optional  | The maximum width in pixels supported for this image. Requests for images sizes with width greater than this may not be supported. _MUST_ be specified if `maxHeight` is specified. |
+| `maxHeight` | Optional  | The maximum height in pixels supported for this image. Requests for images sizes with height greater than this may not be supported. If `maxWidth` is specified and `maxHeight` is not, then clients should infer that `maxHeight = maxWidth`.  |
+| `maxArea`   | Optional  | The maximum area in pixels supported for this image. Requests for images sizes with width\*height greater than this may not be supported. |
```

Comment:
> images sizes with width\*height greater than this may not be supported?

width\*height ?

Reply:
Oh, escaping?

Reply:
Yep, escaped to avoid it trying to parse as a bold with no end

Reply:
Oh, and:

> may not be supported

do we really mean that they _might_ be? Why can't we say _MUST_ not be supported?

Reply:
True. I think they're not supported. Will fix.